Mathlib/RingTheory/Finiteness/Cardinality.lean

```diff
@@ -37,6 +38,12 @@ lemma exists_fin' [Module.Finite R M] : ∃ (n : ℕ) (f : (Fin n → R) →ₗ[
   refine ⟨n, Basis.constr (Pi.basisFun R _) ℕ s, ?_⟩
   rw [← LinearMap.range_eq_top, Basis.constr_range, hs]
 
+/-- A finite module can be realised as a quotient of `Fin n → R` (i.e. `R^n`). -/
+theorem exists_fin_equiv (R M : Type*) [Ring R] [AddCommGroup M] [Module R M] [Module.Finite R M] :
```

Comment:
I think the name should at least mention quotient.

Reply:
it is now `exists_fin_quot_equiv`